addon/components/stripe-element.js

```diff
@@ -64,6 +64,25 @@ export default Component.extend({
 
   setEventListeners() {
     let stripeElement = get(this, 'stripeElement');
+    stripeElement.on('ready',   (event) => this.sendAction('ready', stripeElement, event));
```

Comment:
The changes to this file need to be removed because they are unrelated to what this PR is trying to do - i.e. add options to the `init` method on the service.